docs/cloud/satellites/gha-runners.md

```diff
@@ -0,0 +1,168 @@
+# Satellites as GHA runners (**experimental**)
+
+Earthly Satellites embed now a GitHub self-hosted runner, so they can directly pull jobs from GHA without the need of an intermediate runner.
+
+This self-hosted runner comes with the Earthly CLI preinstalled, and it's configured to use the Satellite BuildKit instance, so GHA jobs will share the same Satellite cache than the traditional Satellite builds.
+
+Also, notice that the self-hosted runner can run any arbitrary GHA job, not necessarily an Earthly command, and given that it runs within the Satellite, it benefits from its persistent local storage (see ["Persistent Folders" section below](#persistent-folders)) across builds.
+
+## Early access
+This feature is in closed-beta at the moment. You can request early access through support@earthly.dev
+
+## Configuration
+Once your organization is allowed to use this feature, next step is to set the integration. You have two possible options here:
```

Comment:
this feels a bit wordier than needed. maybe one can just say:

> Once approved for the closed-beta, you can enable satellite-based GHA workers for specific GitHub Repositories or for all repositories in your organization.

and then remove the unordered list.